### ISSUE TITLE
[Pools of Aggonar] Slime Covered Corpse

### ISSUE THREAD

**Issue by snipz69** [ccshiro/cc-buglist#60]
Came across this quest objective NPC, though he was not exactly a corpse, http://i.imgur.com/mPMPMuG.jpg. After the server crashed at ~21:00, I went back to check and he was in his Blizzlike state again (dead).
http://www.wowhead.com/npc=20158/slime-covered-corpse http://www.wowhead.com/quest=9374/arelions-journal

**Comment by ccshiro** [comment]
Restarted the server several times and tried various things, couldn't get the bug to happen.

**Comment by nz54445** [comment]
I tried to reproduce this but i just can't. I aggroed nearly every mob i could see around, jumped around so he would get hit by spells, tried casting my spells on him, feared mobs through him, tried fearing him, stuck around for about 4-5 server crashes he was not up ever. I wonder if snipz69 clicked to see if the guy was actually a corpse but was just playing a different animation.

Anyway, that's my two cents on this one. Maybe it should be moved to low priority or closed until someone can reproduce? Maybe it was fixed along the way.
